### PR TITLE
Chore/tidy operator token

### PR DESCRIPTION
**Pull request by awalias**
purpose is to remove operator token from non-multi-instance deployments

### CODE REVIEW

File: api/middleware.go [hunk 1]
```diff
@@ -91,7 +92,7 @@ func (a *API) loadInstanceConfig(w http.ResponseWriter, r *http.Request) (contex
 	claims := NetlifyMicroserviceClaims{}
 	p := jwt.Parser{ValidMethods: []string{jwt.SigningMethodHS256.Name}}
 	_, err := p.ParseWithClaims(signature, &claims, func(token *jwt.Token) (interface{}, error) {
-		return []byte(a.config.OperatorToken), nil
+		return []byte(config.JWT.Secret), nil
```
Comment (awalias) [inline api/middleware.go:95]: actually not needed since we're not utilizing multi-instance mode anyway

**Comment by awalias** [comment]
tested locally with facebook, twitter, google, and invite

File: api/middleware.go (hunk 1 above)
Comment (kangmingtay) [reply api/middleware.go:95]: yeah but i think we may need it in the future when we move gotrue to multi-tenant 😅

File: api/middleware.go [hunk 2]
```diff
@@ -150,38 +151,19 @@ func (a *API) limitHandler(lmt *limiter.Limiter) middlewareHandler {
...
-	if err == nil {
-		return c, nil
-	}
-
-	if t == "" {
+	ctx := req.Context()
+	t, err := a.extractBearerToken(w, req)
+	if err != nil || t == "" {
 		return nil, err
 	}
```
Comment (kangmingtay) [inline api/middleware.go:160]: I think we need to do a `withAdminUser` here to set the admin user to the context. Seems like it was originally being set [here](https://github.com/supabase/gotrue/blob/e3d1d95f7f963041fb1d78e89b9a289e442717ba/api/middleware.go#L166) in `extractOperatorRequest`.

The [adminUserUpdate](https://github.com/supabase/gotrue/blob/e3d1d95f7f963041fb1d78e89b9a289e442717ba/api/admin.go#L95) / Create / Delete methods call `getAdminUser` to retrieve the admin user from the context.

Comment (awalias) [reply api/middleware.go:160]: my assumption was that in a Supabase context:
1) [`token != a.config.OperatorToken`](https://github.com/supabase/gotrue/blob/e3d1d95f7f963041fb1d78e89b9a289e442717ba/api/middleware.go#L163) would always resolve to true
2) [this error ](https://github.com/supabase/gotrue/blob/e3d1d95f7f963041fb1d78e89b9a289e442717ba/api/middleware.go#L164)was returned
3) and [silently ignored](https://github.com/supabase/gotrue/blob/e3d1d95f7f963041fb1d78e89b9a289e442717ba/api/middleware.go#L171)

i.e. the only way `withAdminUser` could be set, would be if the user passed `OperatorToken` as the bearer token

Comment (kangmingtay) [reply api/middleware.go:160]: hmm not sure if i missed something but i don't see `withAdminUser` being called anywhere? so we no longer set the adminUser in the context? 

I was thinking we should do the following after line 159
```
ctx = withAdminUser(req.Context(), &models.User{ID: uuid.Nil, Email: "operator@supabase.com"})
```

Comment (kangmingtay) [reply api/middleware.go:160]: oh i just realised that we set the context [here](https://github.com/supabase/gotrue/blob/1cfdfbaa529d1fba9964c93529f371c88f0f80f5/api/auth.go#L36) if the authentication is valid, so there's no need to do what i suggested above

**Review by kangmingtay — APPROVED** [review]

**Pull request merged**